modules/masterbar/masterbar.php

```diff
@@ -563,6 +563,20 @@ public function add_my_sites_submenu( $wp_admin_bar ) {
 			) );
 		}
 
+		// Site Preview
+		if ( is_admin() ) {
+			$wp_admin_bar->add_menu( array(
+				'parent' => 'blog',
+				'id'     => 'site-preview',
+				'title'  => __( 'Site Preview' ),
```

Comment:
Thanks @eliorivero, added it in d5e87c85a42dc764f3f3dbf401ee0c2e8688d286 (dropped `esc_html` since I don't think we need it here). I've also changed the title following the discussion in D6074-code.